dashboard/charts.py

```diff
@@ -34,9 +36,10 @@ def get_most_popular_artists_chart(artists: DataFrame) -> alt.Chart:
 @st.cache_data
 def get_most_popular_tags_chart(tags: DataFrame) -> alt.Chart:
     """Returns a bar chart of popular tags and their sales."""
-
+    tags["total_sales"] = tags["total_sales"].astype(int)
```

Comment:
How would you handle cases when the columns don't exist or fail to convert?

Reply:
I am doing an inner join so the dataframe will contain that column